wasp/draw565.py

```diff
@@ -329,6 +329,15 @@ def string(self, s, x, y, width=None, right=False):
         if width:
             self.fill(bg, x, y, rightpad, h)
 
+    def char_size(self, char):
+        """Get the size of a specific character in the current font.
+
+        :param char: Character to get the size of.
+        :returns     Tuple with  (width, height) of character.
+        """
```

Comment:
The auto build is failing for this pull request and is reporting problems with this docstring. The missing : after returns is likely to cause this.